staging/src/k8s.io/api/testdata/HEAD/apps.v1.StatefulSet.yaml

```diff
@@ -991,6 +991,11 @@ spec:
     spec:
       accessModes:
       - ',躻[鶆f盧詳痍4''N擻搧'
+      allocatedResources:
+        limits:
+          弊þ­奿ÆŁĪŀc=Ƨz鈡煰敹x: "659"
```

Comment:
I assume this is to verify that unicode strings are considered valid input?

Reply:
I think so, but these are generated stuff that gets updated when I ran `make update`